pkg/kubelet/kuberuntime/kuberuntime_container.go

```diff
@@ -1147,6 +1147,11 @@ func (m *kubeGenericRuntimeManager) computeInitContainerActions(pod *v1.Pod, pod
 						changes.InitContainersToStart = append(changes.InitContainersToStart, i)
```

Comment:
Done.

Reply:
I guess instead of `continue` we need a `break` statement here. With continue statement, unit tests started failing: https://prow.k8s.io/view/gs/kubernetes-ci-logs/pr-logs/pull/128367/pull-kubernetes-unit/1851699075053785088